chrome_webstore_description.txt

```diff
@@ -1,4 +1,4 @@
-New in 0.2.5:
```

Comment:
Can you please add this change a new "New in 0.2.6 (not released)" section?

Reply:
I updated the file.